app/views/report/_report_modal.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal fade" id="reportModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button class="close" data-dismiss="modal" type="button">×</button>
+        <h3 class="modal-title"><%= t("report.modal.title", pod_name: pod_name) %></h3>
+      </div>
+      <div class="modal-body">
+        <% if AppConfig.settings.terms.enable? %>
+          <p><%= t("report.modal.report_only_if_violated_terms") %> <%= link_to(t("report.modal.terms_link", pod_name: pod_name), terms_path, target: "_blank") %></a>.</p>
+        <% end %>
+        <p><%= t("report.modal.you_can_block") %></p>
+        <form id="report-content-form" accept-charset="UTF-8">
+          <p><label for="report-reason-field"><%= t("report.modal.reason_label") %></label></p>
+          <p><textarea id="report-reason-field" class="form-control" placeholder="<%= t('report.modal.placeholder') %>" autofocus required></textarea></p>
```

Comment:
See also https://github.com/diaspora/diaspora/pull/8239#discussion_r956804649 and https://github.com/diaspora/diaspora/pull/8239#discussion_r956804826

Reply:
I think the whole interface can be improved, but because the work on this already started in https://github.com/diaspora/diaspora/pull/8239 I don't think I should change it in this MR, it will create conflicts. I think it's fine to merge it even if everything is displayed on one line, it doesn't break the UI: 
![image](https://user-images.githubusercontent.com/930064/198007759-898620c1-c20c-43a1-a83d-ea0da810c5b8.png)
If you disagree then I would suggest to have only one merge request for this and https://github.com/diaspora/diaspora/pull/8239 so that we work on the same branch and are not overlapping the work.

Reply:
Well, it needs to be fixed somewhere, and I would have thought it fits better to this PR, because as soon as this is merged people are probably starting to send multi-line reports which then aren't rendered correctly. But if you rather want to fix it in the other PR, that's fine for me too, as long as both are in the same release (so 1.0 probably). But the current state is pretty "broken".

Reply:
As said I would like to do this in https://github.com/diaspora/diaspora/pull/8239 to avoid conflicts. I will take care of it once this PR is merged if Thorsten is not around.